src/screens/Asset/ReceiveModal.js

```diff
@@ -258,3 +338,57 @@ const WarningText = styled(Text)`
   color: ${({ theme }) => theme.colors.secondaryText};
   text-align: center;
 `;
+
+const ActivateText = styled(Text)`
+  ${fontStyles.small};
+  color: ${({ theme }) => theme.colors.link};
+  text-align: center;
+`;
+
+const DeployText = styled(Text)`
+  ${fontStyles.small};
+  color: ${({ theme }) => theme.colors.secondaryText};
+  justify-content: center;
+  margin: ${spacing.medium}px 0 0;
+  padding: 0 ${spacing.medium}px 0 ${spacing.medium}px;
+`;
+
+const IconContainer = styled.View`
+  align-items: center;
+  justify-content: center;
+`;
+
+const Container = styled.View`
+  background-color: ${({ theme }) => theme.colors.basic080};
+  flex-direction: row;
+  margin: ${spacing.small}px 0;
+  padding: ${spacing.medium}px;
+  border-radius: 22.5px;
```

Comment:
Do we have this as a standard somewhere? If not could you add it to the theme please?

Reply:
do you mean the border radius? because margin and padding is already with the spacing variable which is apt to this situation and i dont think we use this in any other styling

Reply:
Done

Reply:
Thanks for that and the extras too